File: plugin/testDataSwift/shared/basics/IfExpressions.shared.swift
//Package: com.test
//Converted using Khrysalis2

import Foundation



public func main() -> Void {
    var number: Int32 = 0
    number += {if true {
        return 1
    } else {
        return 0
    }}()
    var letCheckValue: String?  = nil
    number += { () in 
        if let it = (letCheckValue) {
            return 2
        }
        return nil
    }() ?? 0
    print(number)
}
 

File: plugin/testDataOutputTs/shared/basics/IfExpressions.shared.ts
//Package: com.test
//Converted using Khrysalis2


export function main(): void {
    let aNumber : number = 0 
    ;
    aNumber += if ( true ) 1 else 0;
    const letCheckValue : string | null = null;
    aNumber += letCheckValue?.let{ 2
     } ?? 0;
    console.log(aNumber)
}
 
